### PR TITLE
Add MDFT (WIP)

### PR DESCRIPTION
**Pull request by itsdfish**

### CODE REVIEW

**Comment by kiante-fernandez** [comment]
Was the equation you were thinking of implementing this one? 

$$V(t+1, i) = \theta V(t, i) + (1 - \theta) \left( \sum_{j=1}^n W_{ij} \phi_{ij}(t) C_j(t) \right)$$

where 

- $V(t, i)$: represents the preference state for response option \( i \) at time \( t \). 

- $\theta$: leakage parameter 

- $W_{ij}$: The weight matrix element indicating the influence of option \( j \) on option \( i \). 

- $\phi_{ij}(t)$: valence function at time \( t \), indicating the relative attractiveness of comparing option \( i \) to option \( j \) 

- $C_j(t)$: perceptual input for option \( j \) at time \( t \)

**Comment by itsdfish** [comment]
Hi @kiante-fernandez. Yes. I am implementing something similar to the equation above.  It is from Equation 2 in _Multialternative Decision Field Theory: A Dynamic Connectionist Model of Decision Making_. What is committed has not been run or tested, and is almost certainly incorrect. I made some changes and wanted to see whether there were performance issues, and decided to run it remotely because my computer is sensitive to heat.

File: src/MDFT.jl [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+σ = 1.0
+τ = 0.30
+Δt = .001
+
+dist = MDFT(; ν, α, β, λ, τ, σ, Δt)
+choices,rts = rand(dist, 500)
+```
+# References
+
+Usher, M., & McClelland, J. L. (2001). The time course of perceptual choice: The leaky, competing accumulator model. Psychological Review, 108 3, 550–592. https://doi.org/10.1037/0033-295X.108.3.550
```
Comment (DominiqueMakowski) [inline src/MDFT.jl:45]: is that the right ref for MDFT?

Comment (itsdfish) [reply src/MDFT.jl]: Good catch. I copied over code from the LCA. The doc strings will be fixed in the next commit now that the code is producing the correct output.

**Pull request merged**